README.Rmd

```diff
@@ -40,8 +40,104 @@ You can install the development version of elicitr from GitHub with:
+
+```{r}
+my_elicitation <- elic_cont_start(var = c("var1", "var2", "var3"),
+                                  var_types = "ZNp",
+                                  elic_types = "134",
+                                  experts = 6,
+                                  title = "Elicitation example")
+```
+###
+```{r, echo = FALSE}
+my_elicitation
+```
+
+Once the metadata has been added to the `elic_cont`object, the data of the first round of elicitation can be added with the function `elic_cont_add_data()`;
+
+```{r}
+my_elicitation <- elic_cont_add_data(my_elicitation,
+                                    data_source = round_1,
+                                    round = 1)
+```
+
+The information message confirms that the data for the first round has been added to the metadata from a `data.frame`. Besides `data frames`, elicitr also allows users to add data from `.csv` or `.xlsx` files, and from Google Sheets.
+
+If you conducted a second round of elicitation, it can be added to the `elic_cont` object after the first round has been added:
+
+```{r}
+my_elicitation <- elic_cont_add_data(my_elicitation,
+                                    data_source = round_2,
+                                    round = 2)
+```
+
+To keep the anonymity of experts, their names are converted to short sha1 hashes and saved in the `id` column. These are then used to match the expert’s answers in the two rounds.
+
+The function `elic_cont_get_data()` retrieves data from an `elicit` object. It is possible to get the whole dataset of a given round, or extract only the data for a given variable, variable type, or elicitation type.
+
+```{r}
+elic_cont_get_data(my_elicitation,
+                   round = 1,
+                   var = "all")
+```
+
+Finally, data can be plotted using the function `elic_cont_plot()`. This function plots data belonging to a given round and for a given variable.
+
+```{r cont_plot}
+elic_cont_plot(my_elicitation,
+               round = 2,
+               group = TRUE,
+               var = "var3",
+               xlab = "Variable 3",
+               ylab = "Experts")
+```
+
+The variable plotted in this case is the result of a four points elicitation, where minimum and maximum estimates, best guess, and expert confidence is provided. In the plot, the best guess is represented with a dot, and the range between minimum and maximum estimates is represented with a line. Expert estimates are represented in purple, while the group's mean is represented in orange.
+
+The message printed here informs users that the minimum and maximum value given by experts have been rescaled using their provided confidence level. Users can choose how they want to rescale these values by providing a `scale_conf` argument.
+
+The scaled minimum and maximum values are obtained with:
+
+$minimum = best\ guess - (best\ guess - minimum)\frac{scale\_conf}{confidence}$
+$maximum = best\ guess + (maximum - best\ guess) \frac{scale\_conf}{confidence}$
```

Comment:
Here I would leave only the first sentence, the rest is too detailed for a readme file. Probably the formula should be moved to the function documentation.